stone/target/js_helpers.py

```diff
@@ -45,8 +48,53 @@ def fmt_obj(o):
         return json.dumps(o, indent=2)
 
 
-def fmt_type(data_type):
-    return _type_table.get(data_type.__class__, 'Object')
+def _create_user_defined_type(data_type, user_defined_types, process_doc, extra_parameters):
+    """
+    Generates a JSDoc @typedef for a user generated type (struct or union type).
+    """
+    type_docstring = '/**'
+    type_docstring += '\n * %s' % data_type.doc if data_type.doc else ''
```

Comment:
You should use the `emit_wrapped_text` helper to bound line length. I think we've standardized on 80 characters. @RileyTomasek to confirm.